src/components/Nav/components/Search/SearchBox.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from 'styled-components';
+import { flex } from '../../../../styles/mixin';
+
+export default function SearchBox({ filteredList, keyword }) {
+  return (
+    <SearchBoxDiv>
+      {filteredList.length >= 1 && keyword !== '' && (
+        <>
+          {filteredList.map(product => (
+            <div key={product.id}>
+              <img src={product.img} alt="Product Img" />
+              <div>
+                <p>{product.name}</p>
+                <p>{product.price}원</p>
+              </div>
+            </div>
+          ))}
+        </>
+      )}
+    </SearchBoxDiv>
+  );
+}
+
+const SearchBoxDiv = styled.div`
+  position: fixed;
+  width: 380px;
+  top: 7%;
+  left: 35%;
+  margin-top: 5px;
+  box-shadow: 2px 3px 5px 0px gray;
+  background-color: white;
+
+  div {
+    ${flex('space=between', 'center', '')}
+    padding: 5px 10px;
+    z-index: 1;
+
+    img {
+      width: 40px;
+      height: 40px;
+    }
+
+    div {
+      ${flex('', 'flex-start', 'column')}
+      gap: 5px;
+    }
+  }
```

Comment:
* nesting 보다는, 각각의 컴포넌트가 독립적인 스타일을 갖고 있는 컴포넌트로 활용될 수 있게 
styled component로 선언 해 주세요.

* 여기서 만약 nesting 한다면, div,img 까지는 가능한데, img화 같은 라인에 있는 div는 nested에서 빼고 component로 스타일링 해주시는게 좋겠습니다.

* 그리고 style을 import해서 사용하는거로 convention이 맞춰졌다면 여기있는 SearchBoxDiv도 파일로 분리되야 하겠습니다.
* 
```suggestion
  const Div = styled.div` 
    ${flex('space=between', 'center', '')}
    padding: 5px 10px;
    z-index: 1;

    img {
      width: 40px;
      height: 40px;
    }
 `
 const AnotherDiv =  styled.div`
   ${flex(null, 'flex-start', 'column')}
      gap: 5px;
    `
```

Reply:
프론트엔드끼리 얘기해보았는데, nav나 main같은 컴포넌트는 style을 import해주기로 했고, nav나 main 안에서 사용되는 컴포넌트(searchbox)는 같은 파일 내에서 style 작성하기로 했습니다!!

Reply:
- 그리고 style을 import해서 사용하는거로 convention이 맞춰졌다면 여기있는 SearchBoxDiv도 파일로 분리되야 하겠습니다
=> 프론트엔드끼리 얘기해보았는데, nav나 main 안에서 사용하는 컴포넌트의 경우 style을 줄 것이 많지 않아서 한 파일 내에서 작성하기로 했습니다~~